flyway/sql/V1__create_tables.sql

```diff
@@ -59,3 +59,7 @@ CREATE TABLE IF NOT EXISTS `ethnicity`
   `identifier` VARCHAR(100) NOT NULL
 ) CHARACTER SET utf8 COLLATE utf8_general_ci;
 
+-- Alter answer table to create composite unique for (visitor id ,question id)
+
+ALTER TABLE `mpop_database`.`answer` DROP INDEX `visitor_question_id`,
```

Comment:
Instead of ALTER TABLE, let's change change the CREATE statement. We'll all have to re-run the database creation script.